recipe/meta.yaml

```diff
@@ -6,23 +6,33 @@ package:
   version: {{ version }}
 
 source:
-  url: https://github.com/ccache/ccache/releases/download/v{{ version }}/ccache-{{ version }}.tar.gz
-  sha256: 9b61f7fab2ec15305ee8382b03365503232dfb04b57414e7b0433768d530fa8a
+  url: https://github.com/ccache/ccache/releases/download/v{{ version }}/ccache-{{ version }}.tar.gz  # [not win]
+  sha256: 9b61f7fab2ec15305ee8382b03365503232dfb04b57414e7b0433768d530fa8a  # [not win]
+  # git_url: https://github.com/ccache/ccache.git  # [win]
+  # git_tag: v3.7.7  # [win]
+  # patches:  # [win]
+  #   - 0001-Visual-C-C-compiler-support.patch  # [win]
+  git_url: https://github.com/cristianadam/ccache.git  # [win]
```

Comment:
Why do you need a repository for Windows?

Is this because this repo adds support for Windows?

Has the author tried to merge it upstream?

Reply:
Yes to both. The Windows build is also based on an earlier version v.3.7.1). I will be adding patches for Windows issues too which I will try to push upstream once the fork has merged properly. Everyone is interested in the merge happening AFAICT. I don't propose we release official packages until after that either. If people want to help test in the meantime I'll be making my own builds on anaconda.org/rdonnelly